apps/webstore/src/app/pages/auth/collection/collection.component.ts

```diff
@@ -1,7 +1,8 @@
 import { HttpErrorResponse } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
-import { DesignTypeEnum, IDesign } from '@interfaces';
+import { ErrorlogPriorityEnum, DesignTypeEnum, IDesign } from '@interfaces';
```

Comment:
```suggestion
import { ErrorlogPriorityEnum, IDesign } from '@interfaces';
```

Reply:
There seems to be a warning that the designTypeEnum is not in use, so i guess it should be removed.

Reply:
Resolved in [9d5ad6a](https://github.com/treecreate/webstore/pull/347/commits/9d5ad6af64b711ed5dddb6f6f032eebbf2ba2b39)